index.html

```diff
@@ -101,7 +101,7 @@ <h2 class="subtitle">Apache Kafka is a powerful, scalable, fault-tolerant distri
             <div>Join data.</div>
           </div>
           <ul>
-            <li><a href="join-a-stream-to-a-table/ksql.html">Join a stream and a table</a></li>
+            <li><a href="join-a-stream-to-a-table/ksql.html">Join a stream and a lookup table</a></li>
```

Comment:
@awalther28 for symmetry to the other KTs listed below this one, what do you think about removing `lookup` from here?  (Leaving `lookup` in the KT itself)

Reply:
It did low key break my heart to ruin to the symmetry of:
`Join a stream and a table`
`Join a stream and a stream`
`Join a table and a table`
If this KT was going through the "how to's" of the connector, I'd feel strongly about keeping the word lookup there but it doesn't 🙂  thus it has been removed.